Lib/urllib/parse.py

```diff
@@ -897,14 +904,11 @@ def quote_from_bytes(bs, safe='/'):
         # Normalize 'safe' by converting to bytes and removing non-ASCII chars
         safe = safe.encode('ascii', 'ignore')
     else:
-        safe = bytes([c for c in safe if c < 128])
```

Comment:
This is a de-optimization. List comprehension is faster than generator expression.

Reply:
sad and understandable, but it appears true.  i restored them.